src/firebase/userDb.js

```diff
@@ -12,13 +12,13 @@ import property_json from "./data/propertyData";
 import user_json from "./data/userData";
 
 export async function getAllUsers() {
-	const allUsers = "user_mock_data";
-	const querySnapshot = await getDocs(collection(db, allUsers));
-	let users = [];
-	querySnapshot.forEach((doc) => {
-		users.push(doc.data());
-	});
-	return users;
+  const allUsers = "user_mock_data";
```

Comment:
can this be updated to `users`? means @bjtrieu doesn't need to make another PR just to do this. otherwise looks good, thanks for taking care of this!

Reply:
Fixed!